server/tests/range_test.py

```diff
@@ -18,53 +18,227 @@
+                "Count_Person_60To61Years",
+                "Count_Person_62To64Years",
+                "Count_Person_65To74Years",
+                "Count_Person_65OrMoreYears",
+                "Count_Person_75OrMoreYears",
+                "Count_Person_5To17Years",
+                "Count_Person_18To24Years",
+                "Count_Person_25To34Years",
+                "Count_Person_35To44Years",
+                "Count_Person_45To54Years",
+                "Count_Person_45To54Years",
+                "Count_Person_55To64Years",
+                "Count_Person_65To74Years",
+            ],
+            "geoId/12345": [
+                "Count_Person_5To17Years",
+                "Count_Person_18To24Years",
+                "Count_Person_25To34Years",
+                "Count_Person_35To44Years",
+                "Count_Person_45To54Years",
+                "Count_Person_45To54Years",
+                "Count_Person_55To64Years",
+                "Count_Person_65To74Years",
+            ]
+        }
         expected = {
-            'Count_Person_5To17Years': ['Count_Person_5To17Years'],
-            'Count_Person_18To24Years': ['Count_Person_18To24Years'],
-            'Count_Person_25To34Years': ['Count_Person_25To34Years'],
-            'Count_Person_35To44Years': ['Count_Person_35To44Years'],
-            'Count_Person_45To54Years': ['Count_Person_45To54Years'],
-            'Count_Person_55To64Years': [
-                'Count_Person_55To59Years', 'Count_Person_60To61Years',
-                'Count_Person_62To64Years'
+            'country/USA': {
+                'Count_Person_5To17Years': ['Count_Person_5To17Years'],
+                'Count_Person_18To24Years': ['Count_Person_18To24Years'],
+                'Count_Person_25To34Years': ['Count_Person_25To34Years'],
+                'Count_Person_35To44Years': ['Count_Person_35To44Years'],
+                'Count_Person_45To54Years': ['Count_Person_45To54Years'],
+                'Count_Person_55To64Years': [
+                    'Count_Person_60To61Years', 'Count_Person_55To59Years',
+                    'Count_Person_62To64Years'
+                ],
+                'Count_Person_65To74Years': ['Count_Person_65To74Years'],
+                'Count_Person_75OrMoreYears': ['Count_Person_75OrMoreYears']
+            },
+            'geoId/12345': {
+                'Count_Person_5To17Years': ['Count_Person_5To17Years'],
+                'Count_Person_18To24Years': ['Count_Person_18To24Years'],
+                'Count_Person_25To34Years': ['Count_Person_25To34Years'],
+                'Count_Person_35To44Years': ['Count_Person_35To44Years'],
+                'Count_Person_45To54Years': ['Count_Person_45To54Years'],
+                'Count_Person_65To74Years': ['Count_Person_65To74Years']
+            }
+        }
+        assert lib_range.aggregate_age_stat_var(input) == expected
+
+    def test_eu_place_age(self):
+        input = {
+            "country/USA": [
```

Comment:
maybe update the dcid for this example so it's clearer?

Reply:
done